src/Shell/Task/ModelTask.php

```diff
@@ -337,7 +340,7 @@ public function findHasMany($model, array $associations)
                         'alias' => $otherModel->alias(),
                         'foreignKey' => $fieldName
                     ];
-                } elseif ($otherTable === $tableName && $fieldName === 'parent_id') {
+                } elseif ($otherTableName === $tableName && $fieldName === 'parent_id') {
```

Comment:
This should not be here and is due to a f*ckup on my side. Once the other PR is cleared I will revisit this and do it through a git feature branch.